src/main/java/edu/depaul/cdm/se452/d2l_mock/course/CalendarWidget.java

```diff
@@ -0,0 +1,11 @@
+package edu.depaul.cdm.se452.d2l_mock.course;
+
+import org.springframework.stereotype.Component;
+
+@Component
```

Comment:
Do we need this class for MS2 which only relates to persistence?

Reply:
Technically no. It's not even working 100% yet so we can not merge it for now.